packages/voici/src/main.ts

```diff
@@ -166,10 +169,10 @@ async function main() {
   const kernelspecs = await jupyterLiteServer.resolveRequiredService(
     IKernelSpecs
   );
-
   const serviceManager = jupyterLiteServer.serviceManager;
+
   const app = new VoiciApp({
-    serviceManager: serviceManager as any,
+    serviceManager: serviceManager,
```

Comment:
Maybe something more succinct:

```suggestion
    serviceManager,
```

Reply:
Thanks!